projects/dev-schematics/src/add-test-outlets/files/test-outlets/test-outlet.module.ts.template

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
```

Comment:
Can we somehow directly pull those files from storefrontapp?
I would love to maintain this in one place. Not in app and schematics.

Reply:
Sadly, we can't. Schematics operate on an active directory when applied. Also renaming files to contain '*.template' isn't accidental, because Schematics can't copy / move / do whatever with files if they're not flagged in any way. I will try to dig into this issue once more

Reply:
But could we in schematics build script create those template files dynamically into final bundle from the repo?

Reply:
I like this suggestion to write a script for this. Maybe the script can do the following:
- copy these files to a certain path in the dev-schematics projects (this folder should be git-ignored)
- append `.template` to all files that were copied

It can also be very useful for iterative development to clean this directory before copying files.